docker/README.rst

```diff
@@ -224,3 +224,44 @@ DOCKER_HQ_OVERLAYFS_METACOPY=[ on | **off** ]
    (performance optimization, has security implications). (Default: "off")
 
 See ``.travis.yml`` for environment variable options used on Travis.
+
+
+Run containers with Podman instead of Docker
+============================================
+
+Podman 4.3 or later can be used to run HQ containers. Unlike docker, podman is
+daemonless and runs containers in rootless mode by default. Podman 4.x is
+available on recent versions of Ubuntu. Older versions, such as Ubuntu 22.04,
+require `a third-party package repository <https://podman.io/docs/installation#debian>`_.
+
+
+Install Podman
```

Comment:
I know you mention it just above here, but if someone jumps to this section immediately and is running 22.04, it might be good to flag that they will need to follow the steps in https://podman.io/docs/installation#debian

Reply:
I think if they do that without reading the paragraph immediately above they get what they deserve (RTFM, yada yada).

It should be possible to get it working with older versions back to at least Podman 3.4. At least that's the first version of Podman that I used. Older versions may require more system tweaks.